diff --git b/public_html/.htaccess a/public_html/.htaccess
index 6f9123d..fe3a5c3 100644
--- b/public_html/.htaccess
+++ a/public_html/.htaccess
@@ -180,3 +180,13 @@ AddEncoding gzip svgz
   # Disable Proxy header, since it's an attack vector.
   RequestHeader unset Proxy
 </IfModule>
+
+# Apache protection
+<If "%{HTTP_HOST} == 'test.yourdomain.org'">
+  <IfModule mod_auth_basic.c>
+    AuthName "Restricted Area"
+    AuthType Basic
+    AuthUserFile "/home/user/.htpasswds/passwd"
+    require valid-user
+  </IfModule>
+</If>
